src/main/drivers/compass/compass_ak8963.h

```diff
@@ -17,4 +17,50 @@
 
 #pragma once
 
+// This sensor is also available also part of the MPU-9250 connected to the secondary I2C bus.
+
+// AK8963, mag sensor address
+#define AK8963_MAG_I2C_ADDRESS          0x0C
+#define AK8963_Device_ID                0x48
+
+// Registers
+#define AK8963_MAG_REG_WIA              0x00
+#define AK8963_MAG_REG_INFO             0x01
+#define AK8963_MAG_REG_ST1              0x02
+#define AK8963_MAG_REG_HXL              0x03
+#define AK8963_MAG_REG_HXH              0x04
+#define AK8963_MAG_REG_HYL              0x05
+#define AK8963_MAG_REG_HYH              0x06
+#define AK8963_MAG_REG_HZL              0x07
+#define AK8963_MAG_REG_HZH              0x08
+#define AK8963_MAG_REG_ST2              0x09
+#define AK8963_MAG_REG_CNTL1            0x0a
+#define AK8963_MAG_REG_CNTL2            0x0b
+#define AK8963_MAG_REG_ASCT             0x0c // self test
+#define AK8963_MAG_REG_I2CDIS           0x0f
+#define AK8963_MAG_REG_ASAX             0x10 // Fuse ROM x-axis sensitivity adjustment value
+#define AK8963_MAG_REG_ASAY             0x11 // Fuse ROM y-axis sensitivity adjustment value
+#define AK8963_MAG_REG_ASAZ             0x12 // Fuse ROM z-axis sensitivity adjustment value
+
+#define READ_FLAG                       0x80
+
+#define ST1_DATA_READY                  0x01
```

Comment:
`STATUS1_` and `STATUS2_` are better prefixes. Abbreviation just makes the code less easy to read.